modules/core/keeper/keeper.go

```diff
@@ -33,7 +33,6 @@ type Keeper struct {
 	ConnectionKeeper *connectionkeeper.Keeper
 	ChannelKeeper    *channelkeeper.Keeper
 	PortKeeper       *portkeeper.Keeper
-	Router           *porttypes.Router
```

Comment:
Removing this is API breaking, so we should document it in the migration docs of v9. I will also change the PR name to reflect this.